apps/dot-voting/app/components/EmptyFilteredVotes.js

```diff
@@ -24,14 +24,14 @@ function EmptyFilteredVotes({ onClear }) {
           src={noResultsSvg}
           alt="No results"
         />
-        <h3
+        <h1
           css={`
             font-size: 28px;
             color: ${theme.content};
           `}
         >
           No results found.
-        </h3>
+        </h1>
```

Comment:
I don't think having this be a heading seems useful. If you think of the headings as being the outline of a document, I don't think it's useful to have "No results found" show up in the outline.

Let's maybe switch to a `<Text size="xxlarge">` while we're here

Reply:
I'll ask design team for opinion :)